### ISSUE TITLE
Argument for dependencies, dev-dependencies, or both

### ISSUE THREAD

**Issue by tekdj7** [MousaZeidBaker/poetryup#37]
Argument for dependencies, dev-dependencies, or both

**Comment by MousaZeidBaker** [comment]
Could you elaborate more and give a detailed description of the issue?

**Comment by tekdj7** [comment]
When doing the `--latest` argument, it updates both dependencies and dev-dependencies.  It be nice, if we had an argument to state which dependencies we wanted to update, `dependencies, dev-dependencies, or both`?

**Comment by jaklan** [comment]
@MousaZeidBaker it would be also nice to be able to specify single package to update instead of running the command against all dependencies

**Comment by MousaZeidBaker** [comment]
@jaklan single packages can be updated using `poetry` itself

**Comment by jaklan** [comment]
@MousaZeidBaker in what way? Afaik I need to run `poetry remove some_package1 some_package2 && poetry add some_package1 some_package2` to update `pyproject.toml` as well.

**Comment by MousaZeidBaker** [comment]
@jaklan Yes that`s the way to go, remove it and then add it with `poetry`

**Comment by jaklan** [comment]
@MousaZeidBaker that's why I think it could be cover as well by `poetryup`, so `poetryup some_package1` would be translated under the hood to the above command. Thanks to that you can introduce `poetryup` in a project as the only tool to update dependencies, with no need to mix `poetryup` with pure `poetry` for that goal.

**Comment by MousaZeidBaker** [comment]
@jaklan I see what you mean, will add it to the list of features.

**Comment by XF-FW** [comment]
> @MousaZeidBaker in what way? Afaik I need to run `poetry remove some_package1 some_package2 && poetry add some_package1 some_package2` to update `pyproject.toml` as well.

Just chiming in, to say that, for that usecase this works and updates `pyproject.toml` as well:
`poetry add some_package1@latest some_package2@latest`
